src/System.Private.Xml/tests/XmlResolver/System.Xml.XmlResolver.Tests/DummyStream.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+
+namespace System.Xml.XmlResolver.Tests
+{
+    internal class DummyStream : MemoryStream
+    {
+        internal DummyStream(byte[] data) : base(data) { }        
+        public override bool CanSeek => false;
```

Comment:
You might also wanna override Seek and throw some kind of exception there  (or something along the lines of  `Assert.True(false, "shouldn't get here");`)